### DISCUSSION TITLE
unexplained pop-up window in orcaslicer

### DISCUSSION THREAD

**Discussion by kinosA1** [OrcaSlicer/OrcaSlicer#4699] [Q&A]
Hola Reciba una advertencia cuando lamino una figura y no entiendo por qué. antes de que no lo consegue y no cambié nada, ¿podrías explicarme cómo resolverlo? el mensaje es este: (solo la primera longitud de extrusión para el modelo de compensación de relleno de área pequeña puede ser 0) Tengo un K1 con la raíz realizada en líquido 
solo me pasa desde que actualicé a la última versión 
![Captura de pantalla 2024-03-24 a las 22 40 07](https://github.com/SoftFever/OrcaSlicer/assets/124101376/82f88c65-c09c-4e4f-9302-d4ba2e990789)

**Comment by TexZeTech** [comment]
To preface this I don't speak Spanish this is just a quick google translate from English to Spanish.


"Only the first extrusion length for small area infill compensation model can be 0"

"Sólo la primera longitud de extrusión para el modelo de compensación de relleno de área pequeña puede ser 0"


I don't know what would have caused that error off the top of my head. 

--> No sé qué habría causado ese error que se me viene a la cabeza.


Could you save and upload your work so others can take a look and also link to the model you are printing off?

--> ¿Podrías guardar y cargar tu trabajo para que otros puedan echarle un vistazo y también vincularlo al modelo que estás imprimiendo?
